app/src/main/res/values/strings.xml

```diff
@@ -22,4 +22,6 @@
     <string name="send_button">SEND</string>
     <string name="permission_required">Permission required</string>
     <string name="txt_turn_on_bluetooth">Turn on Bluetooth and try again</string>
+    <string name="empty_message">Please enter the message to be displayed</string>
```

Comment:
enter a message*

Reply:
Hey, I'm talking about empty_message string. But you change the another one. Sorry for the confusion.